atoma-service/src/handlers/chat_completions.rs

```diff
@@ -194,7 +194,8 @@ pub struct ChatCompletionsOpenApi;
 #[instrument(
     level = "info",
     skip(state, payload),
```

Comment:
This is not directly included in this PR but should these be `skip_all` as well?

Reply:
Yes, good catch !